src/components/EachContinent.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { FaArrowLeft } from 'react-icons/fa';
+import NavBar from './shared/NavBar';
+import { fetchRegionDetails } from '../redux/countries';
+
+const EachContent = () => {
+  const regionDetails = useSelector((state) => state.entities.countries);
+
```

Comment:
- Tweak this part to implement the filter functionality as shown above in the previous comment.